dist/vue-strap.min.js

```diff
@@ -1,6 +1,6 @@
unction(n){w.push.apply(t,n.querySelectorAll(e))}),l(t,this.owner)}},{key:"findChildren",value:function(e){var t=this;return e?this.find(e).filter(function(e){return t.includes(e.parentElement)}):l(this.map(function(e){return e.children}))}},{key:"forEach",value:function(){for(var e=arguments.length,t=Array(e),n=0;n<e;n++)t[n]=arguments[n];return w.forEach.apply(this,t),this}},{key:"includes",value:function(e,t){return~this.indexOf(e,t)}},{key:"map",value:function(){for(var e=arguments.length,t=Array(e),n=0;n<e;n++)t[n]=arguments[n];var r=w.map.apply(this,t);return r.some(function(e){return o(e)||i(e)})?l(r,this):r}},{key:"parent",value:function(){return l(this.map(function(e){return e.parentNode}),this)}},{key:"pop",value:function e(t){"number"!=typeof t&&(t=1);for(var n=[],e=w.pop.bind(this);t--;)n.push(e());return c(n,this)}},{key:"push",value:function(){for(var e=this,t=arguments.length,n=Array(t),r=0;r<t;r++)n[r]=arguments[r];return w.forEach.call(n,function(t){if(!o(t))throw x;~e.indexOf(t)||w.push.call(e,t)}),this}},{key:"shift",value:function(e){"number"!=typeof e&&(e=1);for(var t=[];e--;)t.push(w.shift.call(this));return 1==t.length?t[0]:c(t,this)}},{key:"slice",value:function(){for(var e=arguments.length,t=Array(e),n=0;n<e;n++)t[n]=arguments[n];return c(w.slice.apply(this,t),this)}},{key:"splice",value:function(){for(var e=arguments.length,t=Array(e),n=0;n<e;n++)t[n]=arguments[n];for(var r=2,i=t.length;r<i;r++)if(!o(t[r]))throw x;return w.splice.apply(this,t),this}},{key:"unshift",value:function e(){for(var t=this,e=w.unshift.bind(this),n=arguments.length,r=Array(n),i=0;i<n;i++)r[i]=arguments[i];return w.forEach.call(r,function(n){if(!o(n))throw x;~t.indexOf(n)||e(n)}),this}},{key:"addClass",value:function(e){return this.toggleClass(e,!0)}},{key:"removeClass",value:function(e){return this.toggleClass(e,!1)}},{key:"toggleClass",value:function(e,t){var n=void 0===t||null===t?"toggle":t?"add":"remove";return"string"==typeof e&&(e=a(e)),this.each(function(t){var r=a(t.className);e.forEach(function(e){var t=~r.indexOf(e);t||"remove"===n||r.push(e),t&&"add"!==n&&(r=r.filter(function(t){return t!==e}))}),r=s(r),r?t.className=r:t.removeAttribute("class")}),this}},{key:"get",value:function(e){var t=[];return this.each(function(n){null!==n&&(n=n[e]),t.push(n)}),l(t,this)}},{key:"set",value:function(e,t){return e.constructor===Object?this.each(function(t){if(t)for(var n in e)n in t&&(t[n]=e[n])}):this.each(function(n){e in n&&(n[e]=t)}),this}},{key:"call",value:function(){for(var e=arguments.length,t=Array(e),n=0;n<e;n++)t[n]=arguments[n];var r=w.shift.call(t),o=[],i=!0;return this.each(function(e){e&&e[r]instanceof Function?(e=e[r].apply(e,t),o.push(e),i&&void 0!==e&&(i=!1)):o.push(void 0)}),i?this:l(o,this)}},{key:"item",value:function(e){return c([this[e]],this)}},{key:"on",value:function(e,t,n){if("string"==typeof e&&(e=a(e)),!this||!this.length)return this;if(void 0===n&&(n=t,t=null),!n)return this;var r=n;return n=t?function(e){var n=c(t,thi
```

Comment:
@danielbrzn Please update your jQuery via `npm install`.

Reply:
No need, I [update dist and docs](https://github.com/MarkBind/vue-strap/commits/master) for every release.